Remotion/Validation/UnitTests/ValidationRuleCollectorBaseTest.cs

```diff
@@ -74,6 +76,29 @@ public void AddRule_ForProperty_OneRule ()
       Assert.That(resultAsComponentRuleBuilder.PropertyMetaValidationRuleCollector, Is.SameAs(metaValidationPropertyRule));
     }
 
+    [Test]
+    public void AddRule_ForPropertyViaPropertyInformation_OneRule ()
+    {
+      var info = PropertyInfoAdapter.Create(MemberInfoFromExpressionUtility.GetProperty(_firstNameExpression));
```

Comment:
naming: please use `property` or `propertyInfo`. Just `info` hides the relevant information.